### DISCUSSION TITLE
@ApplicationModuleTest should use org.springframework.test.context.cache

### DISCUSSION THREAD

**Discussion by midovlvn** [spring-projects/spring-modulith#178] [Ideas]
Actually @ApplicationModuleTest does not use context cache across tests. The cache increase performance and promotes best practice. 

Without the cache developers who want fast execution must put all verifications into one, big test class.

**Comment by odrotbohm** [comment]
Can you elaborate why you think it doesn't use the cache? It uses `SpringBootTestContextBootstrapper` and thus should produce a proper key to leverage the default context caching implemented by the test context framework.
